app/models/deep-link-handle-store/deep-link-handle-store.ts

```diff
@@ -43,14 +43,14 @@ const BaseModel = types
 
     const followAppReferrer = flow(function*(likerID: string) {
       if (!likerID) return
+      // Create barebone Liker model with Liker ID
       const liker = self.creatorsStore.createCreatorFromLikerID(likerID)
-      if (!liker.isFollowing) {
+      // Fetch the Liker's details for validation
+      yield liker.fetchDetails()
+      // Validate the Liker by checking the existence of the liker's display name
+      if (liker.displayName && !liker.isFollowing) {
```

Comment:
In API I always return a displayName via `displayName || id`
Not really robust check but I guess ok, if its app's behaviour to not stub displayName with id in any circumstances

Reply:
For this case, I want to ensure the existence of the Liker, if the response status is 404, the displayName should remain an empty string.